composer.json

```diff
@@ -44,4 +44,4 @@
         "test": "vendor/bin/phpunit",
         "test-ci": "vendor/bin/phpunit --coverage-text --coverage-clover=build/coverage.xml"
     }
-}
```

Comment:
Not really sure what happend here. I used the GitHub UI to apply these changes. No IDE involved

Reply:
probably the end of line is always added in the GitHub UI